### DISCUSSION TITLE
You are marked as spam, and therefore cannot authorize a third party application.

### DISCUSSION THREAD

**Discussion by Snickers-khalil** [community/community#101687] [New to GitHub]
Hi GitHub Support Team,

I hope this message finds you well. I am reaching out to report an issue I am facing with my GitHub account.

I recently attempted to authorize a third-party application, but I encountered a problem. It seems that my account has been marked as spam, preventing me from completing the authorization process. This restriction is hindering my ability to use certain features and services associated with the third-party application.

I would appreciate it if you could investigate this matter and assist in resolving the issue. If there are any specific actions or information required from my end, please let me know, and I will promptly provide the necessary details.

Thank you for your attention to this matter. I look forward to a prompt resolution so that I can continue to utilize GitHub seamlessly.

Best regards,

**Comment by ghostinhershell** [comment]
Hi @Snickers-khalil,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
